README.md

```diff
@@ -48,7 +48,7 @@ The component accepts the following props:
 |  **`onAfterPrint?`**   | `function` | Callback function that triggers after the print dialog is closed regardless of if the user selected to print or cancel                                                                                      |
 |  **`onPrintError?`**   | `function` | Callback function (signature: `function(errorLocation: 'onBeforePrint' | 'onBeforeGetContent', error: Error)`) that will be called if there is a printing error serious enough that printing cannot continue. Currently limited to Promise rejections in `onBeforeGetContent` or `onBeforePrint`. Use this to attempt to print again. `errorLocation` will tell you in which callback the Promise was rejected.                                                                                     |
 | **`removeAfterPrint?`** | `boolean`  | Remove the print iframe after action. Defaults to `false`.                                                                                                 |
-|    **`pageStyle?`**    | `string | function`   | We set some basic styles to help improve page printing. Use this to override them and provide your own. If given as a function, it must return a `string`                                                                                               |
+|    **`pageStyle?`**    | `string \| function`   | We set some basic styles to help improve page printing. Use this to override them and provide your own. If given as a function, it must return a `string`                                                                                               |
```

Comment:
From the most recent reply in the upstream thread, it seems the current behavior is correct actually, and npm and the others have it wrong. I still think most people are going to be reading the README on npm, but we should for sure have a solution that works on both.

It's a bit hacky, but maybe let's change this to `string or function` just to get around the problem for now? Happy to hear a different solution if you think of something better

Reply:
Sure, I'm ok with that :+1:

Thinking "out loud": `|` is closer to how TypeScript would represent the types... but not everyone would understand what `|` means, so I think an `or` works well.